src/AcceptanceTests/DelayedDelivery/SendOnly_Sending_when_sender_and_receiver_are_properly_configured.cs

```diff
@@ -0,0 +1,121 @@
+﻿namespace NServiceBus.AcceptanceTests.DelayedDelivery
+{
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using EndpointTemplates;
+    using NServiceBus;
+    using NUnit.Framework;
+    using System;
+    using System.Threading.Tasks;
+
+    public class SendOnly_Sending_when_sender_and_receiver_are_properly_configured : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_deliver_message_if_below_threshold()
+        {
+            var payload = "some payload";
+            var delay = TimeSpan.FromSeconds(2);
+
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<SendOnlySender>(b => b.When(async (session, c) =>
+                {
+                    var sendOptions = new SendOptions();
+                    sendOptions.DelayDeliveryWith(delay);
+
+                    await session.Send(new DelayedMessage
+                    {
+                        Payload = payload
+                    }, sendOptions);
+
+                    c.SentAt = DateTime.UtcNow;
+                }))
+                .WithEndpoint<Receiver>()
+                .Done(c => c.Received)
+                .Run();
+
+            Assert.GreaterOrEqual(context.ReceivedAt - context.SentAt, delay, "The message has been received earlier than expected, we're so good!");
+            Assert.AreEqual(payload, context.Payload, "The received payload doesn't match the sent one. BAD BAD BAD");
+        }
+
+        [Test]
+        public async Task Should_send_message_if_above_threshold()
+        {
+            var payload = "some payload";
+            var delay = TimeSpan.FromMinutes(16);
```

Comment:
Why minutes? The idea was that we make the threshold configurable and then we can do any arbitrary seconds delay over fifo

Reply:
We want to avoid configuring transport to accomodate test values.
In addition to that, we'd not be testing anything above 15 minutes anyways using ATTs, so better to verify that send is a valid send. Send and delivery with over 15 minutes would need to be a manual smoke test.

Reply:
Well, we could configure the `*-delays.fifo` queue to have a smaller delay than 15 minutes, say 15 seconds and use that as a test baseline